spacy/tests/doc/test_span.py

```diff
@@ -178,6 +178,7 @@ def test_span_string_label(doc):
     assert span.label == doc.vocab.strings["hello"]
 
 
+@pytest.mark.xfail(reason="label is not writable")
```

Comment:
Instead of xfailing, we should probably explicitly test for the error here. For instance:

```python
span = Span(doc, 0, 1)
with pytest.raises(NotImplementedError):
    span.label_ = "hello"
```